pom.xml

```diff
@@ -2300,6 +2300,22 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>flink-1.18</id>
+            <properties>
+                <flink.version>1.18.0</flink.version>
+            </properties>
+            <repositories>
```

Comment:
Since the Flink 1.18 is currently in the release candidate stage, the repository will be removed after the stable binary release.

Reply:
it's also required to override `flink.archive.mirror`, it is used to download flink runtime tgz for integration testing